VantageCloudLakeUseCases/content/ExternalObjectStore/README.md

```diff
@@ -126,11 +145,11 @@ Before running the following examples, replace the following fields in the examp
 * [optionally] *AccessKey* : from the Access Key for your bucket - Secret Access Key example: wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY
 
 ### Example 1 
-This first example will select all rows in local sample_csv_local to copy the dataset to the object store's *sample1* partition:
+This first example will select all rows in sample_sensor to copy the dataset to the object store’s *sample1* partition: 
```

Comment:
Simply as:

This example selects all rows in the **sample_sensor** table to copy the dataset to the object store's *sample1* partition: